xtask/src/launcher.rs

```diff
@@ -122,26 +126,14 @@ fn run_variant(variant: LauncherMode) -> Step {
         "build Oak Functions enclave binary",
         &variant.enclave_crate_path(),
     )];
-    // If we want to run in an VMM, we need to do some extra steps:
-    // 1. build the stage0 BIOS image
-    // 2. build the enclave shim that wraps the kernel
-    // 3. embed the built oak_functions binary in the .payload section of the shim
-    // and then we can start QEMU with stage0 as the firmware and the merged enclave_bin as the ELF
-    // binary to load.
+    // If we want to run in an VMM, we need three binaries:
+    // 1. the stage0 BIOS image,
+    // 2. the enclave shim that wraps the kernel,
```

Comment:
I guess this is part of future cleanup work, but we don't really need a shim anymore. The restricted kernel itself can just become a binary.

Reply:
Conceptually I agree. In practice, until we figure out how to make binaries targeting x86_64-unknown-none part of the workspace, I'd keep it separate, purely for development ergonomics.